### DISCUSSION TITLE
How to update

### DISCUSSION THREAD

**Discussion by pettywise** [sherlock-project/sherlock#1769] [Q&A]
Maybe I'm just an idiot and looking completely over something I'm doing wrong but I cannot seem to update sherlock correctly. I keep triggering this code:

Update Available!
You are running version 0.14.2. Version 0.14.3 is available at https://github.com/sherlock-project/sherlock
ERROR:  Problem parsing json contents at 'https://raw.githubusercontent.com/sherlock-project/sherlock/master/sherlock/resources/data.json':  Missing attribute 'username_unclaimed

if someone can post a simple step by step of commands to input to properly install updates/json file I would be beyond grateful.

**Comment by murfdogprod** [comment]
"Git pull"

**Comment by ykcgw** [comment]
I don't get your answer...
